File: Assets/Scripts/ClientInfo.cs
using System.Collections;
using System.Collections.Generic;
using System.Net.Sockets;
using UnityEngine;

public class ClientInfo
{
	public Socket socket;
	public Queue<string> sendQueue, receiveQueue;
	public string Name;
	public Room room;

	public ClientInfo(Socket _socket, Queue<string> _sendQueue, Queue<string> _receiveQueue) {
		socket = _socket;
		sendQueue = _sendQueue;
		receiveQueue = _receiveQueue;
		Name = "";
		room = null;
	}
}

File: Server/ClientInfo.java
package server;

import java.net.Socket;
import java.util.Queue;

public class ClientInfo {
	public Socket socket;
	public Queue<String> sendQueue;
	public String name;
	public int udp_port;
	public Room room;
	
	public ClientInfo(Socket _socket, Queue<String> _sendQueue) {
		socket = _socket;
		sendQueue = _sendQueue;
		name = "";
		udp_port = 0;
		room = null;
	}
}
